projects/distributed-tracing/src/shared/dashboard/data/graphql/waterfall/trace-waterfall-data-source.model.ts

```diff
@@ -60,6 +60,14 @@ export class TraceWaterfallDataSourceModel extends GraphQlDataSourceModel<Waterf
     this.specificationBuilder.attributeSpecificationForKey('errorCount')
   ];
 
+  protected readonly logEventSpecifications: Specification[] = [
+    this.specificationBuilder.attributeSpecificationForKey('traceId'),
```

Comment:
nit: do we actually need the spanid and trace id in these? isn't it already available in the span parent?

Reply:
yeah, we don't need that actually.
thank you for pointing that out.